include/deal.II/grid/grid_tools.h

```diff
@@ -1032,6 +1032,52 @@ namespace GridTools
   ( const MeshType                                                                    &mesh,
     const std::function<bool (const typename MeshType::active_cell_iterator &)> &predicate );
 
+  /**
+   * Compute a collection of bounding boxes so that all cells for which the given predicate is true, are
+   * completely enclosed in at least one of the bounding boxes.
+   *
+   * For each cell at a given refinement level containing active cells for which predicate is true,
+   * the function creates a bounding box of its children for which predicate is true.
```

Comment:
I don't understand. Active cells don't have children.

Reply:
he's referring to `cells at a given refinement level`. Those *do* have children in general....

Reply:
I see. So would "For every active cell on a level equal or larger than @p refinement_level and with @p predicate true, the function creates a bounding box." be correct?

Reply:
Not really... if all children of "cell" (which is not active) have property we just use "cell" and not its children to create the bounding box (this is done by compute_cell_predicate_bounding_box).

Reply:
So it is always a single box for each cell on the given level covering all children with the predicate? Then I totally misunderstood.